runtime/src/serde_snapshot.rs

```diff
@@ -705,6 +705,12 @@ where
 
     let mut measure_notify = Measure::start("accounts_notify");
 
+    let IndexGenerationInfo { accounts_data_len } = accounts_db.generate_index(
```

Comment:
This is functionally okay. The problem is with it will slow overall snapshot restore as before we are doing notify_account_restore_from_snapshot and  accounts_db.generate_index in parallel. With this it will be make the two slow operations serial. Are there any other alternatives?